docs/nddata/subclassing.rst

```diff
@@ -156,16 +156,16 @@ on the class or the instance::
     >>> ndd = NDDataPower([1,2,3])
 
     >>> # using it on the instance with one operand
-    >>> ndd.pow(3)
+    >>> ndd.pow(3)  # doctest: +SKIP
```

Comment:
Why are these three doctest skipped?

Reply:
It turns out the numpy repr formatting also prints the dtype, but only on Windows. The skip avoids an error on the windows test.

Reply:
Is there no way to use ellipsis? Do you still have the error log on Windows? Seems excessive to skip a valid doctest just because of one OS/arch.

p.s. Maybe we have to fast track dropping doctest on certain platforms but that is a discussion for another day.

Reply:
Here's the failure: https://github.com/astropy/astropy/actions/runs/5859004595/job/15884145980#step:10:1734

Are you suggesting something like this?
```diff
-     >>> ndd.pow(3)  # doctest: +SKIP
-    NDDataPower([ 1,  8, 27])

+    >>> ndd.pow(3)  # doctest: +ELLIPSIS
+    NDDataPower([ 1,  8, 27]...)
```
Personally I think ellipsis is more confusing.

Reply:
https://github.com/scientific-python/pytest-doctestplus#doctest-dependencies allows conditional skipping though I never tried it with platform check. Maybe we can do that. If not, please open a follow-up issue to un-skip this later because I feel like this should not be skipped if possible.

Also I am going to cc @mhvk here since this might add to the things he has to handle over at #15065

Reply:
I'm happy to open an issue. It'd be great to get it working correctly everywhere, but it seems like the easiest way to do that now is to add conditional repr formatting in astropy to clean up the platform differences.

Reply:
Think I got it in [0e57b49](https://github.com/astropy/astropy/pull/14438/commits/0e57b49c3439129221c35a6a6037b6ba21c908ac)!